app/javascript/components/automated-workflows/summary.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Loading, Tabs, Tab, CodeSnippet,
+} from 'carbon-components-react';
+import AWSSfnGraph from '@tshepomgaga/aws-sfn-graph';
+import MiqStructuredList from '../miq-structured-list';
+import { automatedWorkflowData } from './automated-workflows-dummy-data';
+import '@tshepomgaga/aws-sfn-graph/index.css';
+
+const AutomatedWorkflowSummary = ({ recordId }) => {
```

Comment:
Similar to the comment above, let's leave out the word `Automated` or `Embedded`, so this should just be `WorkflowSummary` and the file path should be `javascript/components/workflows/summary.jsx`

Same comment goes for the other .jsx files and directories.